client/js/shout.js

```diff
@@ -382,6 +382,28 @@ $(function() {
 		}
 	}
 
+	if ($.cookie("timestamp")) {
```

Comment:
Use the existing `settings` cookie object, it will be easier to switch to another storage method in the future.